htdocs/core/boxes/box_factures_fourn_imp.php

```diff
@@ -101,6 +101,7 @@ public function loadBox($max = 5)
 			$sql .= " AND f.entity = ".$conf->entity;
 			$sql .= " AND f.paye = 0";
 			$sql .= " AND fk_statut = 1";
+			$sql .= " AND f.date_lim_reglement IS NOT NULL";
```

Comment:
If an unpaid invoice has no dead line, it is always an unpaid invoice, isn't it ?
So why discarding it ?

Reply:
@eldy Yes trus, the real bug (warning php) was fix in dol_print_date